sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -1073,7 +1073,7 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, val isDriver: B
   protected lazy val blockManager = env.blockManager
   protected lazy val shouldFallThroughOnEverything = {
     val fallThroughReasons = new ListBuffer[String]()
-    if (GpuShuffleEnv.isExternalShuffleEnabled) {
+    if (!rapidsConf.isMultiThreadedShuffleManagerMode && GpuShuffleEnv.isExternalShuffleEnabled) {
       fallThroughReasons += "External Shuffle Service is enabled"
     }
     if (GpuShuffleEnv.isSparkAuthenticateEnabled) {
```

Comment:
@jbrennan333 same goes for "Spark authentication is enabled", the MT shuffle works with auth/encryption.

Reply:
Thanks @abellina! I've added this one as well, and updated the title.